api/project.test.js

```diff
@@ -0,0 +1,53 @@
+import request from "supertest";
+import app from "./src/index.js";
+import { describe, it, expect, beforeAll, afterAll, jest } from "@jest/globals"; // Remove jest import
+import http from "http";
+
+jest.mock("./controllers/swaggerController.js", () => {
+  return function (req, res) {
+    res.send("Swagger documentation mock");
+  };
+});
+
+// jest.mock("./config/db.js", () => ({
```

Comment:
props for doing testing, why is this commented out?

Reply:
cause my tests are failing because of swagger, and something else i dont know yet :(

Reply:
hehe. I believe that if it is swagger you should add it to jest.config to ignore the swagger files. There is no point in testing external lib. In general with unit tests make sure to only select the functions and files you actually need to test. Most of that functionality you can achieve with either jest.config ignore directive and/or mocking :)